core/src/loader.rs

```diff
@@ -2005,6 +2033,16 @@ impl<'gc> Loader<'gc> {
                 .as_container()
                 .unwrap();
 
+            // This isn't completely correct - the 'large_preload' test observes the child
+            // being set after an 'enterFrame' call. However, our current logic should
+            // hopefully be good enough.
+            avm2_stub_method!(
```

Comment:
Might `context_stub` be better for this?

Reply:
This is still associated with `Loader.loadBytes`, even though it's not directly called from that method